admin_guide/pruning_resources.adoc

```diff
@@ -343,3 +344,152 @@ Such tags tend to have just one image in its history, which effectively prevents
 them from ever being pruned.
 xref:../dev_guide/managing_images.adoc#tag-naming[Learn more about _istag_
 naming.]
+
+[[hard-pruning-registry]]
+== Hard Pruning the Registry
+
+The OpenShift Container Registry can accumulate blobs that are not referenced by
+the {product-title} cluster's etcd. The basic xref:pruning-images[Pruning
+Images] procedure therefore is unable to operate on them. These are called
+_orphaned blobs_.
+
+Orphaned blobs can occur from the following scenarios:
+
+- Manually running the `oc delete image <sha256:image-id>` command only removes
+the image from etcd, but not from the registry's storage.
+- Pushing to the registry initiated by *docker* daemon failures causes some blobs
+to get uploaded but the image manifest (which is uploaded as the very last
+component) does not. All unique image blobs become orphans.
+- An image being refused because of quota suffers similarly to the previous
+example.
+- The standard image pruner may delete an image manifest but be interrupted before
+it deletes the related blobs.
+- There was a bug in the registry pruner which failed to remove blobs that were
+supposed to be removed. Image objects referencing them were removed and the
+blobs became orphans.
+// Find this BZ
+
+_Hard pruning_ the registry is a separate procedure from basic image pruning
+that allows you to remove orphaned blobs. You should hard prune if you are
```

Comment:
This sounded at first sounded like the basic image pruning removes the blobs. As in:
"Hard pruning the registry is a separate procedure" from "basic image pruning that allows you to remove orphaned blobs"
Maybe 
_Hard pruning_ the registry, a separate procedure from basic image pruning ,allows you to remove orphaned blobs."
_Hard pruning_ the registry allows you to remove orphaned blobs, but is a separate procedure from basic image pruning "